packages/react/src/components/IonicReactProps.ts

```diff
@@ -1,5 +1,4 @@
 export interface IonicReactProps {
-  class?: string;
```

Comment:
Threading for discussion:

It's possible that there are other developers relying on this `class`. However, using this will cause all Ionic classes to be wiped, so I can't imagine that the experience is very good. 

Reviewers: Do you have any concerns about potential breakages?

From my perspective, React devs should be using `className`, so this `class` never should have been exposed.

Reply:
I don't have any concerns with this breaking change (in a patch). `className` is the standard for all React apps/libraries. `class` is confusing and leads to a degraded experience.